systers_portal/community/models.py

```diff
@@ -13,7 +16,7 @@
 class Community(models.Model):
     """Model to represent Systers community or subcommunity"""
     name = models.CharField(max_length=255, verbose_name="Name")
-    slug = models.SlugField(max_length=150, unique=True, verbose_name="Slug")
+    slug = models.SlugField(max_length=150, unique=True, verbose_name="Slug", editable="False")
```

Comment:
Consider using `blank=True` and drop the editable kwarg as it'll also disable the same from django-admin.